devtools/ssl.ext

```diff
@@ -0,0 +1,7 @@
+authorityKeyIdentifier=keyid,issuer
+basicConstraints=CA:FALSE
+keyUsage = digitalSignature, nonRepudiation, keyEncipherment, dataEncipherment
+subjectAltName = @alt_names
+
+[alt_names]
+DNS.1 = localhost
```

Comment:
@mxm I would say this would give the user the possibility to adapt the `alt_names` for the hosts independently from the python script. what do you think?